tembo-cli/src/cmd/login.rs

```diff
@@ -168,3 +211,92 @@ pub fn update_access_token(
 
     Ok(())
 }
+
+pub fn update_context(org_id: &str, profile_name: &str) -> Result<(), anyhow::Error> {
+    let context_file_path = tembo_context_file_path();
+    let contents = fs::read_to_string(&context_file_path)?;
+    let mut data: Context = toml::from_str(&contents)?;
+
+    let new_env = Environment {
+        name: profile_name.to_string(),
+        target: "tembo-cloud".to_string(),
+        org_id: Some(org_id.to_string()),
+        profile: Some(profile_name.to_string()),
+        set: Some(false),
+        selected_profile: None,
+    };
+    data.environment.push(new_env);
+
+    let modified_contents = toml::to_string(&data)?;
+    fs::write(&context_file_path, modified_contents)?;
+
+    Ok(())
+}
+
+pub fn update_or_create_profile(
```

Comment:
same idea here regarding not replacing the whole content of the file, rather appending to a file.

Reply:
Same here